R/util.r

```diff
@@ -227,6 +227,12 @@ a <- function(text, href) {
   ifelse(is.na(href), text, paste0("<a href='", href, "'>", text, "</a>"))
 }
 
+write_utf8 <- function(..., path, sep = " ") {
```

Comment:
I default to `sep = " "` because it is the default for `cat()` and it was used with default in `cat(contents, file = path)` . I did not want to change that. I can default to `sep = ""` and use `write_utf8(contents, path = path, sep = "")` in `render.r`